tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -876,6 +933,13 @@ public:
 
    void Finalize();
 
+   // Helper functions for RMergeableValue
+   void* GetMergeableValue() {
+      const auto &counts = std::accumulate(fCounts.begin(), fCounts.end(), 0);
```

Comment:
this silently casts `counts` to `int` (classic std::accumulate pitfall), please use `0ull` as init value